src/models/ledger/ripple_state.ts

```diff
@@ -0,0 +1,17 @@
+import { Amount } from "../common";
+
+export interface RippleState {
+  LedgerEntryType: 'RippleState'
+  Flags: number
+  Balance: Amount
+  LowLimit: Amount
+  HighLimit: Amount
```

Comment:
The RippleState documentation only says that fields like Balance can be Amounts (not strings). The Amount type you have here though can be a string. (Not sure if that's just an inconsistency in the documentation though) Should these Amount types be limited to just the IssuedCurrencyAmount object?

Reply:
Link to documentation for reference: https://xrpl.org/ripplestate.html#example-ripplestate-json

Reply:
Good catch!